Sources/Interpreter/BuiltinValue.swift

```diff
@@ -76,7 +76,7 @@ enum BuiltinValue {
 /// Methods to create builtin value from IR constants.
 extension BuiltinValue {
 
-  /// Creates instance of builtin value with integer constant `c`.
+  /// Creates instance of builtin value with `c`.
```

Comment:
I don't know what it means to create an instance “with” something.  “of builtin value” is redunda tn.

 
```suggestion
  /// An instance with value `c`.
```

Maybe?